### ISSUE TITLE
Support 1.8 graceful shutdown

### ISSUE THREAD

**Issue by derekperkins** [Xeoncross/secureserver#1]
https://tylerchr.blog/golang-18-whats-coming/

**Comment by xeoncross** [comment]
Yes, as soon as 1.8 is released several features (including graceful shutdown) are going to be activated in this repository.
